examples/platform/silabs/efr32/rs911x/rsi_if.c

```diff
@@ -55,14 +55,17 @@
 
 #define WFX_QUEUE_SIZE 10
 
-/* Rsi driver Task will use as its stack */
-StackType_t driverRsiTaskStack[WFX_RSI_WLAN_TASK_SZ] = { 0 };
-
-/* Structure that will hold the TCB of the wfxRsi Task being created. */
-StaticTask_t driverRsiTaskBuffer;
-
-/* Declare a variable to hold the data associated with the created event group. */
-StaticEventGroup_t rsiDriverEventGroup;
+#define kDrvTaskSize 1792 // Example size, adjust as needed
```

Comment:
before you were using `WFX_RSI_WLAN_TASK_SZ` define was that also 1792?

Also, you use #define but format as if it is a variable.

constexpr uint32 should be used.  
And for the comment; Users are not expected to go in the platform code source file and edit them directly. If you expect them to change the stack size of the task make it configurable in some way 

```suggestion
constexpr uint32 kDrvTaskSize = 1792;
```